rpcs3/Emu/RSX/VK/VKVertexBuffers.cpp

```diff
@@ -457,12 +459,13 @@ VKGSRender::upload_vertex_data()
 						data_size = converted_buffer.size();
 					}
 
-					auto &buffer = m_attrib_buffers[index];
 
-					buffer.sub_data(0, data_size, data_ptr);
-					buffer.set_format(format);
+					size_t offset_in_attrib_buffer = m_attrib_ring_info.alloc<256>(data_size);
+					void* buf = m_attrib_buffers->map(offset_in_attrib_buffer, data_size);
```

Comment:
I agre with @paulsapps on this one. I think having a buffer update call reduces the verbosity and should help with readablility. Vulkan can be very verbose and hard to follow.